local-env-setup.sh

```diff
@@ -24,7 +24,7 @@ darwin_install_pip3_packages() {
 
 install_go_packages(){
         echo "Installing goavro"
-        go get github.com/linkedin/goavro/v2
+        go install github.com/linkedin/goavro@latest
```

Comment:
Did you mean to make this change? I think we need get here not just install

Reply:
No, this was accidental. Had to make this change to get my dev container to work. [Mailing list thread](https://lists.apache.org/thread/4lp8nqppd2xv8m694oyjzzjqvwtor4gl)

Will revert